app/src/main/res/values/strings.xml

```diff
@@ -1,25 +1,21 @@
 <resources>
     <string name="app_name">MDSApplication</string>
-    <string name="start_playing">Incepe jocul</string>
-    <string name="new_game">Joc nou</string>
-    <string name="new_game_all_caps">JOC NOU</string>
-    <string name="choose_pawn">Alege culoarea pionului</string>
-    <string name="add_members">Adauga membru</string>
+    <string name="start_playing">Start Playing</string>
+    <string name="new_game">New Game</string>
+    <string name="new_game_all_caps">NEW GAME</string>
+    <string name="choose_pawn">Choose the color of your pawn</string>
+    <string name="add_members">Add members</string>
+    <string name="add_meb_of_team">Add the members of the team...</string>
     <string name="assemble_team">Assembling team</string>
-
-    <string name="insertTeamName">Introduceti numele echipei:</string>
-    <string name="insertMemberName">Introduceti un membru:</string>
-    <string name="backWithArrow">&#60; Inapoi</string>
-    <string name="continueWithArrow">Continua &#62;</string>
-    <string name="newGameRo">Joc nou</string>
-    <string name="twoTeams">Doua Echipe</string>
-    <string name="threeTeams">Trei Echipe</string>
-    <string name="fourTeams">Patru Echipe</string>
-
```

Comment:
Consider ca aceste linii nu trebuiau sterse intrucat erau folosite in activitatea NewGame la adaugarea de echipe